cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -36,12 +83,15 @@ func initControlSetShardModeCmd() {
 	flags := setShardModeCmd.Flags()
 	flags.StringSlice(shardIDFlag, nil, "List of shard IDs in base58 encoding")
 	flags.Bool(shardAllFlag, false, "Process all shards")
+
+	modes := make([]string, 0)
+
+	iterateSafeShardModes(func(strMode string) {
+		modes = append(modes, "'"+strMode+"'")
+	})
+
 	flags.String(shardModeFlag, "",
-		fmt.Sprintf("New shard mode keyword ('%s', '%s', '%s')",
-			shardModeReadWrite,
-			shardModeReadOnly,
-			shardModeDegraded,
-		),
+		fmt.Sprintf("New shard mode keyword (%s)", strings.Join(modes, ", ")),
```

Comment:
While we are here, what about removing `keyword` from the message?

Reply:
Done.